lib/exercises.rb

```diff
@@ -1,26 +1,55 @@
 
 # This method will return an array of arrays.
 # Each subarray will have strings which are anagrams of each other
-# Time Complexity: ?
-# Space Complexity: ?
+# Time Complexity: o(n)
+# Space Complexity: o(n)
 
 def grouped_anagrams(strings)
```

Comment:
👍 The time complexity is O(n) if the strings are limited in size, but O(n * m log m) where m is the length of the strings, if the strings aren't limited in size.